core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/hdr/InternalHDRPercentilesRanksTests.java

```diff
@@ -61,4 +66,26 @@ protected void assertReduced(InternalHDRPercentileRanks reduced, List<InternalHD
         return InternalHDRPercentileRanks::new;
     }
 
+    @Override
+    protected NamedXContentRegistry xContentRegistry() {
+        return new NamedXContentRegistry(singletonList(
+                new NamedXContentRegistry.Entry(
+                        Aggregation.class,
+                        new ParseField(InternalHDRPercentileRanks.NAME),
+                        (parser, context) -> ParsedHDRPercentileRanks.fromXContent(parser, (String) context))));
```

Comment:
I think so, but I think it is OK for now and should be easy to change later. I'm in favor of keeping it like this until we have many parsable aggregations.